recipe/run_test.py

```diff
@@ -2,22 +2,30 @@
 # Distributed under the BSD-2-Clause license
 # Copyright (c) 2017, Continuum Analytics, Inc. All rights reserved.
 
-import sys
 import os
+import platform
+import sys
+
 from numba.cuda.cudadrv.libs import test, get_cudalib
 from numba.cuda.cudadrv.nvvm import NVVM
-
+from packaging import version
 
 def run_test():
+    nvvm = NVVM()
+    print("NVVM version", nvvm.get_version())
```

Comment:
Thanks for this improvement/cleanup.

Would it be worth `print`ing the platform and GLIBC version as well? Given how other checks behave below, this seems like useful context

Reply:
Done